src/ui/actions/timeline.ts

```diff
@@ -668,22 +668,18 @@ export function setFocusRegionBeginTime(beginTime: number, sync: boolean): UIThu
 }
 
 export function syncFocusedRegion(): UIThunkAction {
-  return async (dispatch, getState, { replayClient }) => {
+  return async (_dispatch, getState, { replayClient }) => {
```

Comment:
nit/curiosity: why?

Reply:
This is a what we do on the backend repo. It is a way to tag unused variables. This way, the editor (the way mine is set up) won't display the parameter with a fade:

<img width="369" alt="image" src="https://user-images.githubusercontent.com/35577323/209260556-f70374ba-4f01-4e52-aceb-d3137b495000.png">
vs
<img width="182" alt="image" src="https://user-images.githubusercontent.com/35577323/209260503-66e07fa0-76ae-41a2-83b2-0568af6494ce.png">